src/RoleObjects/MultiLineTextBoxData.js

```diff
@@ -329,4 +332,28 @@ export default class MultiLineTextBoxData extends AccessibilityObject {
     event.selectionDirection = evt.currentTarget.selectionDirection;
     this._displayObject.dispatchEvent(event);
   }
+
+  /**
+   * event handler for when the element gets input
+   * @param {Event} evt
+   */
+  _onInput(evt) {
+    this._onChange(evt);
+    this._onSelect(evt);
+  }
+
+  /**
+   * event handler for when keyboard key is lifted Up
+   * @param {Event} evt
+   */
+  _onKeyUp(evt) {
```

Comment:
Recommend seeing https://github.com/CurriculumAssociates/createjs-accessibility/blob/main/src/RoleObjects/GridData.js#L85-L94 and https://github.com/CurriculumAssociates/createjs-accessibility/blob/main/src/RoleObjects/GridData.js#L13-L27 so that you can have an always bound key listener without breaking AccessibilityObject's support for that.